client/src/helpers/fhir/fhir.js

```diff
@@ -4,6 +4,10 @@ import get from 'lodash/get';
 
 const BUNDLE_COUNT = 100
 
+export const EMPTY_NOTE_TEXT = '--' // FHIR doesn't allow empty string
```

Comment:
Strong suggestion but optional: how about you find a .ts file explicitly for notes and comments? I feel like this file is a free for all. Plus, you could use enums and such.